
              <mat-sidenav-content id="page-content"  fxFlex="70%">
          
                   
                  
                    <div id="slider-and-main" fxLayout="column " fxLayout.xs="column" fxLayoutWrap fxLayoutGap="1.5%" fxLayoutAlign="center">
                        <h1>The Mice</h1>
                        <article><p>
                                The Wellcome Trust Sanger Institute currently generates 160 knockout mouse lines a year, all of which are phenotyped by the 3i
                                consortium. One third of the genes are chosen for their likely involvement in human diseases (e.g. as shown in GWAS studies),
                                one third are chosen from suggestions of participating scientists and one third are randomly selected from the International
                                Knockout Mouse Consortium (IKMC) embryonic stem cell pool (see below). Approximately 25% of the knockout lines generated are
                                embryonically lethal and are phenotyped as heterozygotes. Whilst lines homozygous for the targeted alleles present with a higher hit rate,
                                lines assessed in the heterozygous state were still remarkably informative. A characterization of the first 250 knockout mouse lines that completed
                                the Sanger phenotyping pipeline pre-3i has been published by
                                <span>
                            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23870131" target="_blank" >White et al.</a></span>
                                (Cell 2013)
                            </p>
                              <br/>
                            </article>
            
                    
            
                            <mat-accordion>
                                <mat-expansion-panel>
                                    <mat-expansion-panel-header>
                                        <mat-panel-title>
                                                Embryonic stem cells
                                        </mat-panel-title>
            
                                    </mat-expansion-panel-header>
                                    All ES cells are derived from the resources generated by
                                <a href="https://www.mousephenotype.org/about-ikmc" target="_blank">IKMC,</a>
                                        an ongoing effort to create knockout ES cells for every protein-coding gene in the mouse. At the moment, all ES cells used have been generated with traditional techniques. In the future, some of the lines will likely be generated by CRISPR technology.
                                </mat-expansion-panel>
            
                                <mat-expansion-panel>
                                        <mat-expansion-panel-header>
                                            <mat-panel-title>
                                                    Knockout Strategy
                                            </mat-panel-title>
            
                                        </mat-expansion-panel-header>
                                        Most of the knockouts lines are generated by a knock-in first approach (Skarnes et al, Nature 2011). This technique combines the advantages of a reporter-tagged allele with a conditional knockout approach (see figure 1).
            
                                        <img src="assets/images/knockout-first-471x451.png"/>
                                        <p>Historically, approximately 25% of knockout lines at Sanger have been phenotyped as tm1b alleles and 75% as tm1a alleles. An internal study showed that 60% of the tm1a lines were complete knockouts, 16% were hypomorphs and 24% had residual gene activity. By the end of 2014, all lines will be analysed in their tm1b form. The knock-in first approach relies on the identication of a critical exon common to all trascript variants that, when deleted, creates a frame-shift mutation. The critical exon is defined by a computer-assisted procedure and flanked by loxP sites. Upstream, an FRT-flanked lacZ-loxP-neomycin cassette is inserted. The lacZ-loxP-neomycin cassette contains a splice acceptor and also codes for a poly-A tail. In its full-length form (tm1a), the construct therefore generates a null allele by creating a shortened transcript that lacks the critical exon. A lacZ reporter knockout (tm1b) can be generated by exposing mice or cells to Cre recombinase. Alternatively, exposure to Flp recombinase will create a wild type allele (tm1c) which can then be used to create conditional knockout strains by crossing the mice to a tissue-specific Cre-expressing strain. Genes expressed in ES cells are usually targeted with a promoterless construct, genes not expressed with a promoter-driven construct. Promoter-driven constructs carry a slightly higher risk for off-target effects. If the target gene is small and does not contain a critical exon, an exon is split by the targeting cassette imbedded in an artificial intron. More detailed information on the constructs used can be found in the
                                            <span>  <a href="http://www.mousephenotype.org/data/documentation/aboutImpc#howdoesimpcwork" target="_blank">targeting strategy section of the IMPC website.</a></span> </p>
                                        <p></p><p></p>
                                        <p>How to order a knockout mouse or an ES cell line </p>
                                        <p></p>
                                        <p>3i itself does not distribute any materials, but mice and ES cells can be requested via the IMPC for a fee from repositories in Europe or the US. Please search for your gene of interest on the
                                            <span>  <a href="http://www.mousephenotype.org/" target="_blank"> IMPC website to find out which products are available.</a></span></p>
                                   </mat-expansion-panel>
                            </mat-accordion>
            
            
            
            
                        <p></p>
                        <h3 style="font-size:2em;letter-spacing: -1px">Experimental Design and Protocols</h3>
                        <p></p>
                        <h3>Observational screen</h3>
                        <p></p>
                        <p>The observational screen aims to detect any immunological disturbances that occur in key immunological sites in the steady state. Unless stated otherwise, mice are sacrificed at 16 weeks of age.</p>
            
                      <br/>
                      <mat-accordion>
                            <mat-expansion-panel>
                                    <mat-expansion-panel-header>
                                        <mat-panel-title>Flow-cytometric analysis of spleen, mesenteric lymph nodes and bone marrow</mat-panel-title>
                                    </mat-expansion-panel-header>
            
            
                            <p>For each knockout strain, we analyse spleen, mesenteric lymph nodes and bone marrow of 3 female and 3 male mice. The flow-cytometric analysis comprises all major immune cell compartments of these organs.</p>
                            <p></p><p></p>
                            <h4 style="font-size:1em;letter-spacing: -1px">Related Documents</h4>
                            <br/>
                            <ul class="ul-pdf">
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Bonemarrowpreparation-protocol.pdf" target="_blank">Bonemarrowpreparation-protocol</a></span>
                                </li>
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/MLN preparation-protocol.pdf" target="_blank">MLN preparation - protocol</a></span>
                                </li>
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Set-upandpanels.pdf" target="_blank">Set-up and panels</a></span>
                                </li>
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Spleenpreparation-protocol.pdf" target="_blank">Spleen preparation - protocol</a></span>
                                </li>
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Staining-protocol.pdf" target="_blank"> Staining - protocol</a></span>
                                </li>
                            </ul>
                        </mat-expansion-panel>
                        <mat-expansion-panel>
                                <mat-expansion-panel-header>
                                    <mat-panel-title>Flow-cytometric analysis of peripheral blood lymphocytes</mat-panel-title>
                                </mat-expansion-panel-header>
            
                            <p></p>
                            <p>For each knockout strain, we analyse peripheral blood lymphocytes at six week of age of 4 male and 4 female mice. Perioheral blood lymphovytes at 16 weeks of age of 7 female and 7 male is analysed by the Sanger core phenotyping team in addition to the 3i assays. The Sanger core assays also include a haematological analysis. The flow-cytometric analysis comprises all major immune cell compartments in the blood.</p>
                            <p></p>
                            <h4 style="font-size:1em;letter-spacing: -1px">Related Documents</h4>
                            <br/>
                            <ul class="ul-pdf">
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Flow analysisblood- protocol.pdf" target="_blank">Blood Preparation and Staining Protocol</a></span>
                                </li>
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Setup and panels for PBL 16 and 6 weeks_v1.pdf" target="_blank"> Instrument setup and panels </a></span>
                                </li>
                            </ul>
                        </mat-expansion-panel>




                        <h3>Flow Cytometry</h3>
                        <ul>
                            <li>Panels</li>
                            <li>Panel change</li>
                        </ul>                   
                        <p>
                        The project started in late autumn 2013. In mid-2014, we decided to change the original antibody panels to make them more compatible with the
                         IMPC panels and to replace a few antibodies that had not been working as well as expected. For the T cell panel, this change happened on 9th of June 2014, for the B cell panel on 15th of September 2014. Data collected before and after the panel changes is not directly comparable and was analysed separately (constructing separate reference ranges before and after the panel change). 
                        </p>
                        
                        <h3>Flow cytometry: gating</h3>
                        <ul>
                            <li>Gating strategy</li>
                        <li>Manual and automated gating</li>
                        </ul>

                        <p>
                                We were aware at the start of the project that manual gating of all FCS files would be very time-consuming and started immediately to explore different algorithms to automate the gating process.
                                 However, quality control of ongoing phenotyping meant that we still had to gate all files manually before the automated gating was in place. We eventually teamed up with <a href="http://www.terryfoxlab.ca/people-detail/ryan-brinkman/">Ryan Brinkman’s team in Vancouver</a> and developed a semi-supervised gating
                                  algorithm for all our panels except blood after the initial panel change in summer 2014. Getting this right and performing the required validations took longer than anticipated and we therefore ended up with a complete set of manually gated data and an almost complete set of data gated automatically with a semi-supervised algorithm. Both datasets are available in the data download section.
                        </p>

                        <p>
                                Data sent to the IMPC for graphing on the website is based on manual gating. For the consortium publication CITE WHEN OUT, 
                                we reanalysed all available data with the semi-supervised algorithm and refined the hit calling accordingly. Hits displayed online are therefore based on manual gating but verified with automated gating.
                                 We found that automated gating reduced temporal drift in many parameters but had not a huge impact on the hit calling as the reference range only calls extreme phenotypes. 
                                 For further analysis, especially for potential phenotypes that have not made our stringent cut-off criteria, we recommend using the automated data wherever possible.
                        </p>
                        <p>
                                Automated semi-supervised gating
                                Computational gating was performed by Ryan Brinkman’s team with <a href="https://www.bioconductor.org/packages/release/bioc/html/flowClean.html">flowClean</a>, UFO, and <a href="https://www.bioconductor.org/packages/release/bioc/html/flowDensity.html">flowDensity</a>. 
                                FlowClean was used to perform acquisition-based quality checking to remove anomalous events. Files with fewer than 20,000 events were removed from further analysis. UFO was used to identify outlier samples (e.g., batch effects). FlowDensity was used to enumerate cell populations by automating a predefined gating approach using sequential, supervised bivariate gating analysis to set the best cut-off for an individual marker using characteristics of the density distribution.
                                The parameters for each individual cell population were pre-defined once for all files. The automated analysis data was validated against matched manually analysed data.
                                
                        </p>
























                        <mat-expansion-panel>
                                <mat-expansion-panel-header>
                                    <mat-panel-title>Cytotoxic T cell function</mat-panel-title>
                                </mat-expansion-panel-header>
            
                            <p>For each knockout strain, the cytotoxicity of splenic CD8 T cells from 2 female and 2 male mice is assessed in a kill assay using P14 cells.  Cytotoxic T cells are crucial for killing virus-infected and cancerous cells in mice and humans.</p>
                            <p></p>
                            <h4 style="font-size:1em;letter-spacing: -1px">Related Documents</h4><br/>
                            <ul class="ul-pdf">
                                <li><span>
                            <img src="assets/images/pdf.png"  width="20" height="20" />
                            <a href="assets/methods-pdfs/Cytotoxic T cell function - protocol.pdf" target="_blank">Cytotoxic T cell function protocol</a></span>
                                </li>
                            </ul>
                        </mat-expansion-panel>
            
                        <mat-expansion-panel>
                                <mat-expansion-panel-header>
                                    <mat-panel-title>Antinuclear antibodies</mat-panel-title>
                                </mat-expansion-panel-header>
            
                            <p>For each knockout strain, the serum of 3 female and 3 male mice is tested for the presence of antinuclear IgG antibodies. Antinuclear antibodies are a key hallmark of many autoimmune diseases in mice as well as humans. Standard clinical chemistry parameters are assessed in addition by the Sanger core phenotyping team.
                            </p>
                            <h4 style="font-size:1em;letter-spacing: -1px">Related Documents</h4><br/>
                            <ul class="ul-pdf">
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Antinuclear antibodies - protocol.pdf" target="_blank">Antinuclear antibodies protocol</a></span>
                                </li>
                            </ul>
                        </mat-expansion-panel>
            
                        <mat-expansion-panel>
                                <mat-expansion-panel-header>
                                    <mat-panel-title>Microscopic analysis of immune cells in the epidermis</mat-panel-title>
                                </mat-expansion-panel-header>
            
            
                            <p>In order to also include immune cells in the tissue, we analyse the immunological composition of the ear epidermis in 2 female and 2 male mice of each knockout strain, focussing on number, size and shape of Langerhans cells and dendritic epidermal T cells (DETC). The majority of immune cells in mice and humans is found in the tissues where they orchestrate local immune responses, but due to obvious difficulties, they are rarely being looked at in patients.</p>
                            <h4 style="font-size:1em;letter-spacing: -1px">Related Documents</h4><br/>
                            <ul class="ul-pdf">
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Ear epidermal sheet isolation.pdf" target="_blank">Epidermal sheet preparation</a></span>
                                </li>
                                <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Epidermal sheet staining - protocol.pdf" target="_blank">Epidermal sheet staining </a></span>
                                </li>
                            </ul>
                        </mat-expansion-panel>
            
                    </mat-accordion>
            
                        <p></p>
                        <p></p>
                        <h3> Challenges</h3>
                        <p>In order to find defects in the knockout mice that only become apparent when the immune system is challenged, we subject the mice to a broad spectrum of challenges. A Salmonella typhimurium infection as an example of a bacterial infection is carried out by the team of Gordon Dougan at the Wellcome Trust Sanger Institute. For the 3i assays, we chose influenza as a viral infection, Trichuris muris as a parasite infection and DSS colitis as an example of a chemical stressor. Selected strains will furthermore be infected with Mycobacterium tuberculosis.</p>
                      <br/>
            
                    <mat-accordion>
                        <mat-expansion-panel>
                                    <mat-expansion-panel-header>
                                        <mat-panel-title>Salmonella typhimurium infection</mat-panel-title>
                                    </mat-expansion-panel-header>
            
                        <p></p>
                        <p>For each knockout strain, 4 female and 4 male mice are intraperitoneally infected with strain M525, and weight is monitored daily. Half of the mice are sacrificed after 14 days, the other half after 28 days. The liver is collected for bacterial counts and for histology. Wild type littermates from various knockout lines are used as controls.</p>
                        </mat-expansion-panel>
            
                        <mat-expansion-panel>
            
                    <mat-expansion-panel-header>
                        <mat-panel-title>Influenza infection</mat-panel-title>
                    </mat-expansion-panel-header>
            
                        <p></p>
                        <p>For each knockout strain, 3 female and 3 male mice are infected intranasally with strain X-31, and weight is monitored daily. After 10 days, the mice are sacrificed and immune cells from the lung are collected by bronchoalveolar lavage. Wild type littermates from various knockout lines are used as controls.</p>
                        <p></p>
                        <h4 style="font-size:1em;letter-spacing: -1px">Related Documents</h4><br/>
                        <ul class="ul-pdf">
                            <li><span>
                            <img src="assets/images/pdf.png" width="20" height="20" />
                            <a href="assets/methods-pdfs/Bronchoalveloar lavage - protocol.pdf" target="_blank">Bronchoalveloar lavage protocol</a></span>
                            </li>
                        </ul>
            
                    </mat-expansion-panel>
            
                    <mat-expansion-panel>
                        <mat-expansion-panel-header>
                        <mat-panel-title>Trichuris muris infection</mat-panel-title>
                        </mat-expansion-panel-header>
            
                                <p></p>
                                <p>For each knockout strain, 6 female mice are infected with Trichuris muris eggs by oral gavage, and weight is monitored daily. After 33 days, the mice are sacrificed and the cecum is assessed for the presence or absence of worms. Serum is collected for confirmation of productive infection. Wild type littermates from various knockout lines are used as controls.</p>
                                <p></p>
                                <h4 style="font-size:1em;letter-spacing: -1px">Related Documents</h4><br/>
                                <ul class="ul-pdf">
                                    <li><span>
                                    <img src="assets/images/pdf.png" width="20" height="20" />
                                    <a href="assets/methods-pdfs/Trichuris muris infection - protocol.pdf" target="_blank">Trichuris muris infection protocol</a></span>
                                    </li>
                                </ul>
            
                    </mat-expansion-panel>
            
                    <mat-expansion-panel>
                            <mat-expansion-panel-header>
                            <mat-panel-title>DSS colitis</mat-panel-title>
                            </mat-expansion-panel-header>
            
            
                        <p></p>
                        <p>For each knockout strain, 4 male mice receive 1.5% dextran sodium sulphate in drinking water for 7 days, and weight is monitored daily. After 10 days, the mice are sacrificed and samples from the medial and distal gut are collected for histological analysis. Wild type littermates from various knockout lines and wild type mice from a separate colony type colony housed in the same facility are used as controls.</p>
                    </mat-expansion-panel>
            
                    <mat-expansion-panel>
                            <mat-expansion-panel-header>
                            <mat-panel-title>Mycobacterium tuberculosis infection</mat-panel-title>
                            </mat-expansion-panel-header>
            
                        <p></p>
                        <p>For selected knockout strains, 25 male mice are infected with a low dose aerosol of strain H37Rv.  After 24 hours, 3 weeks and 6 weeks, 5 mice each are sacrificed and lung and spleen are collected for bacterial counts. 10 mice are kept for survival analysis.</p>
            
            
                    </mat-expansion-panel>
                    </mat-accordion>
            
                    <h3>Flow Cytometry</h3>
                    <ul>
                        <li>Panels</li>
                        <li>Panel change</li>
                    </ul>

                    <p>After initial phase of the project, we altered flow cytometry panels for compatibility with IMPC panels. Data gathered before and after this panel change is not directly comparable and hence it was analysed separately when assessing phenotypes (we prepared reference range specific for this initial samples). Aforementioned change happened on XXX for B cell panel (spleen and MLN) and on XXX for T, M and BM panel. IMPC panel was added on this date.
                    </p>

                    <h3>Flow Cytrometry: gating</h3>
                    <ul>
                            <li>Gating strategy</li>
                            <li>Manual and Automated Gating</li>
                    </ul>
                    <p>
                            All FCS files were gated manually at the time of acquisition. Non-blood FCS files gathered after panel change were reanalysed afterwards with automated gating by Ryan Brinkman team <a href="">Link todo</a>.  Data presented through the IMPC portal is based on manual gating.
                            We used automated gating whenever possible. For strains where some samples were reanalysed with automated gating and others analysed only with manual gating, we used for each sample the best available data (so preferably automated-gating derived) and compared it to the appropriate reference range. Only then results were aggregated for the strain.
                            
                    </p>
                    <p>
                            In the Data download section we provide information about what sort of gating was used for any specific sample.
                    </p>


            
                    
                    </div>
              </mat-sidenav-content>
              
              <mat-sidenav mode="side" opened fxFlex="28%"  position="end" fxLayout="column " fxLayout.xs="column" fxLayoutGap="10px">
                    
                      <img src="assets/images/Observationalscreen235.jpg"/>
                      <img src="assets/images/Challengescreen235.jpg"/>
                      
                    
                    
              </mat-sidenav>
